lib/queries/get-challenges-by-tag.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
@SimonLab, I agree; I thought that we were after creating more abstract functions in tag-systems plugin... I think it makes sense to do so because I think our queries will be similar for most plugins... Although, maybe trying to make it abstract _to begin with_ is premature optimisation... :/